### PR TITLE
Attribute Precedence Table

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

### Description

Our explanation of attribute precedence is really painful to learn. This PR attempts the attribute precedence list easier to read by using a table.

As 12/31/2020, it looks like:
Screen Shot 2020-12-31 at 3 57 47 PM

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

**Comment by kagarmoe** [comment]
See: https://deploy-preview-2763--chef-web-docs.netlify.app/attributes#attribute-precedence

**Comment by phiggins** [comment]
In the `default` and `override` sections, `role` and `environment` are reversed. Is this correct?

It was present in the original docs page, it just stuck out to me while reviewing.

File: themes/docs-new/layouts/shortcodes/node_attribute_precedence.md [hunk 1]
```diff
@@ -1,27 +1,44 @@
...
+    <td><code>`default`</code></td>
+    <td>1. Cookbook<br>2. Recipe<br>3. Environment<br>4. Role</td>
+  </tr>
+  <tr class="even">
+    <td><code>`force_default`</code></td>
+    <td>1. Cookbook attribute file<br>2. Recipe</td>
+  </tr>
+  <tr class="odd">
+    <td><code>`normal`</code></td>
+    <td>1. JSON file passed with <code>`chef-client -j`</code><br>2. Recipe</td>
```
Comment (IanMadd) [inline themes/docs-new/layouts/shortcodes/node_attribute_precedence.md]: Is `normal` missing Cookbook attribute file?

**Comment by kagarmoe** [comment]
@lamont-granquist Could you give me a code sample for the example in the note? I'm still not clear on how it works.

**Comment by kmf** [comment]
This is so cool, thanks for this ♥️

**Comment by kagarmoe** [comment]
At least one person found this helpful, so let's try it out.

**Review by IanMadd — APPROVED** [review]

**Pull request merged**